bidwire/tests/test_utils.py

```diff
@@ -15,3 +15,8 @@ def test_parallel():
         [(Accumulator.accumulate, 1), (Accumulator.accumulate, 3)])
     assert len(results) == 2
     assert Accumulator.accumulator == 4  # 1 and 3 were accumulated
+
+def _abs_file(filename):
```

Comment:
This method name should _not_ be prefixed with an underscore. The underscore is by convention used to indicate symbols that are meant to be private, which this function is not.

Also, this file is not the right place for this function. I meant to say to start a test-specific `utils` library, which would live in a place like `tests/utils.py`, but this is a test for the main `utils.py` library, which is not the same thing.